CONTRIBUTING.md

```diff
@@ -8,9 +8,14 @@ We welcome contributions to PWA Kit! To ensure that your contributions are addre
 
 This repository is a [monorepo](https://en.wikipedia.org/wiki/Monorepo) that includes the code that powers the PWA Kit. The code is divided into the following packages:
 
--   `pwa`: A set of sample code and tooling for PWA Kit projects called the Retail React App
--   `pwa-kit-create-app`: A tool that generates PWA Kit projects based on the Retail React App
--   `pwa-kit-react-sdk`: A set of components and utilities for PWA Kit projects
+- `template-retail-react-app`: A set of sample code and tooling for PWA Kit projects called the Retail React App
+- `template-express-minimal`: A set of sample code for a minimal express app
+- `template-typescript-minimal`: A set of sample code for a minimal React app that support Typescript 
```

Comment:
```suggestion
- `template-typescript-minimal`: A sample app that can be used as a starting point for projects using TypeScript.
```

Reply:
We should mention that it is still a React app